compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -147,6 +182,12 @@ int entry(int argc, char **argv)
     .required(false)
     .help("Output type of quantized model (uint8 or int16)");
 
+  arser.add_argument(cfg)
+    .nargs(1)
+    .type(arser::DataType::STR)
+    .required(false)
+    .help("Path to the quantization configuration file (string)");
```

Comment:
It might be unnecessary?
```suggestion
    .help("Path to the quantization configuration file");
```